openshift-ci/Dockerfile.deploy

```diff
@@ -1,11 +1,9 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
 ARG BIN_DIR=
-ARG ASSETS_DIR=build/assets
```

Comment:
this is only true for this docker file or should we also align other docker files we maintain ?

Reply:
Yes, we will need to update relevant d/s dockerfile, but it not critical, because it will continue to work also with assets files under the operator image, it will just not use them